### PR TITLE
[l10n] Improve Ukranian (uk-UA) locale for data grid

### PR DESCRIPTION
**Pull request by mkundos**
This PR adds ukUA (Ukrainian) translations to headerFilterOption in DataGrid.

- [+] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

**Comment by MBilalShafi** [comment]
Thank you @mkundos for your contribution. 🙏
Could you run **yarn l10n** and commit the changes to fix the build?

**Comment by mkundos** [comment]
Updated

**Review by MBilalShafi — APPROVED** [review]
Thanks for the update, LGTM

**Comment by mkundos** [comment]
Hi! Are there any estimations when this might be reviewed? We want to release services that use these translations ASAP as it is client-faced and we bought a premium license to have access to all features.

**Comment by LukasTy** [comment]
> Hi! Are there any estimations when this might be reviewed? We want to release services that use these translations ASAP as it is client-faced and we bought a premium license to have access to all features.

Hey @mkundos, thank you for bringing it up. 🙏 
We'll make sure to include it in the next release (today or tomorrow). 😉

File: packages/grid/x-data-grid/src/locales/ukUA.ts [hunk 1]
```diff
@@ -94,33 +94,33 @@ const ukUAGrid: Partial<GridLocaleText> = {
...
+  headerFilterOperatorContains: 'Містить',
+  headerFilterOperatorEquals: 'Дорівнює',
+  headerFilterOperatorStartsWith: 'Починається з',
+  headerFilterOperatorEndsWith: 'Закінчується на',
+  headerFilterOperatorIs: 'Дорівнює',
+  headerFilterOperatorNot: 'Не дорівнює',
+  headerFilterOperatorAfter: 'Більше ніж',
+  headerFilterOperatorOnOrAfter: 'Більше або дорвінює',
+  headerFilterOperatorBefore: 'Менше ніж',
+  headerFilterOperatorOnOrBefore: 'Менше або дорівнює',
```
Comment (cherniavskii) [inline packages/grid/x-data-grid/src/locales/ukUA.ts:114]: These are labels for date operators, shouldn't it be like this?

```suggestion
  headerFilterOperatorAfter: 'Після',
  headerFilterOperatorOnOrAfter: 'Після (включно)',
  headerFilterOperatorBefore: 'Раніше',
  headerFilterOperatorOnOrBefore: 'Раніше (включно)',
```

File: packages/grid/x-data-grid/src/locales/ukUA.ts [hunk 2]
```diff
@@ -94,33 +94,33 @@ const ukUAGrid: Partial<GridLocaleText> = {
...
+  headerFilterOperatorIsEmpty: 'Порожнє',
+  headerFilterOperatorIsNotEmpty: 'Не порожнє',
+  headerFilterOperatorIsAnyOf: 'Будь-що із',
+  'headerFilterOperator=': 'Рівні',
+  'headerFilterOperator!=': 'Не рівні',
```
Comment (cherniavskii) [inline packages/grid/x-data-grid/src/locales/ukUA.ts:119]:
```suggestion
  'headerFilterOperator=': 'Дорівнює',
  'headerFilterOperator!=': 'Не дорівнює',
```

**Review by cherniavskii — COMMENTED** [review]
Дякую!

File: packages/grid/x-data-grid/src/locales/ukUA.ts (hunk 1 above)
Comment (mkundos) [reply packages/grid/x-data-grid/src/locales/ukUA.ts:114]: Звучить цілком логічно, але для орієнтування взяв переклади, що були вище, щоб була однорідність, а змінювати попереднє не став.

90 рядок:
```javascript
  filterOperatorAfter: 'більше ніж',
  filterOperatorOnOrAfter: 'більше або дорівнює',
  filterOperatorBefore: 'менше ніж',
  filterOperatorOnOrBefore: 'менше або дорівнює',
  ```
  Якщо змінювати ці, чи варто змінювати ті, що вже були?

Comment (mkundos) [reply packages/grid/x-data-grid/src/locales/ukUA.ts:114]: переглянув декілька мов, де є переклади обох - в деяких ідентичні рядки (німецька, французька) в інших відрізняються (іспанська).

**Comment by LukasTy** [comment]
Merging it to include it in the release as it seems that the questions were resolved. 👌

**Pull request merged**